### PR TITLE
[Merged by Bors] - doc(Tactic): expand `clear*` docstring

### PR DESCRIPTION
**Pull request by rwst**
Add details and example to `clear*` docstring.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by joneugster** [comment]
Thanks!

maintainer merge

File: Mathlib/Tactic/ClearExcept.lean [hunk 1]
```diff
@@ -16,7 +16,11 @@ open Lean.Meta
 
 namespace Lean.Elab.Tactic
 
-/-- Clears all hypotheses it can besides those provided -/
+/-- Clears all hypotheses it can, besides those provided after a minus sign. Example:
```
Comment (jcommelin) [inline Mathlib/Tactic/ClearExcept.lean:19]:
```suggestion
/-- Clears all hypotheses it can, except those provided after a minus sign. Example:
```

**Review by jcommelin — COMMENTED** [review]
bors d+

**Pull request closed without merging**